src/XrdCms/XrdCmsReq.cc

```diff
@@ -134,17 +134,17 @@ void XrdCmsReq::Reply_Error(int ecode, const char *emsg, int elen)
 // Translate the error name
 //
    switch(ecode)
-         {case ENOENT:       eval = kYR_ENOENT;
-          case EPERM:        eval = kYR_EPERM;
-          case EACCES:       eval = kYR_EACCES;
-          case EIO:          eval = kYR_EIO;
-          case ENOMEM:       eval = kYR_ENOMEM;
-          case ENOSPC:       eval = kYR_ENOSPC;
-          case ENAMETOOLONG: eval = kYR_ENAMETOOLONG;
-          case ENETUNREACH:  eval = kYR_ENETUNREACH;
-          case ENOTBLK:      eval = kYR_ENOTBLK;
-          case EISDIR:       eval = kYR_EISDIR;
-          default:           eval = kYR_EINVAL;
```

Comment:
In this case whatever `ecode` is we set `eval` to `kYR_EINVAL`. This looks wrong, thus probably we forgot to add `break`s otherwise the whole thing can be reduced to a single line: `eval = kYR_EINVAL;`.